src/styles/index.scss

```diff
@@ -1,3 +1,9 @@
-body {
+@import 'variables';
+@import 'card';
+@import 'stars';
+
+body * {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
```

Comment:
it's allowed to move only this rule to `*` selector

Reply:
Did not quite get what was wrong here. If it was about deleting margin values, tests would not pass.